app/templates/events/trackVolunteers.html

```diff
@@ -59,6 +59,7 @@ <h1 class="text-center">{{event.name}}</h1>
         <th>Email</th>
         <th>Phone Number</th>
         <th class="noSort">Completed Trainings?</th>
+        <th class="noSort">Eligible?</th>
```

Comment:
I am not a big fan of the name here, is there anything else we could name it that might be a bit better? Also, I think we might want to take out the punition here, and on the header before.